packages/api/lambdas/sf-event-sqs-to-db-records/utils.js

```diff
@@ -38,24 +37,42 @@ const isPostRDSDeploymentExecution = (cumulusMessage) => {
   }
 };
 
-const hasNoParentExecutionOrExists = async (cumulusMessage, knex) => {
-  const parentArn = getMessageExecutionParentArn(cumulusMessage);
-  if (!parentArn) {
-    return true;
+const getMessageAsyncOperationCumulusId = async (cumulusMessage, knex) => {
+  try {
+    const asyncOperationId = getMessageAsyncOperationId(cumulusMessage);
+    if (!asyncOperationId) {
+      throw new Error('Could not find async operation in message');
+    }
+    const asyncOperation = await knex(tableNames.asyncOperations).where({
```

Comment:
Nit: I think this came up in a prior PR, it still feels odd having this logic in the *api* package, but perhaps that's more that this *lambda* being there feels odd.    Not something to fix for this epic/PR, but bears consideration later if the lambda and/or these utils should be re-orged a bit.